changelog/61725.fixed

```diff
@@ -0,0 +1 @@
+Update docs to replace master_shuffle with random_master.
```

Comment:
Doc changes do not require changelog file changes. I'll go ahead and push a fix for this, but just an fyi for next time.